### ISSUE TITLE
Incorrect description about GitOps on DevOps page

### ISSUE THREAD

**Issue by LinuxSuRen** [kubesphere/website#1847]
Currently (since 3.x), KubeSphere DevOps didn't follow GitOps to implement DevOps flow. You can search GitOps from Google, or check the definition from [www.gitops.tech](https://www.gitops.tech/#what-is-gitops)

![image](https://user-images.githubusercontent.com/1450685/126774567-cb780585-61cd-4a1f-bbec-54349d21590c.png)

I get the screenshot from https://kubesphere.com.cn/devops/

![image](https://user-images.githubusercontent.com/1450685/126774290-839d808d-064b-45da-bac5-b022257e5bd2.png)

![image](https://user-images.githubusercontent.com/1450685/126774369-58a04a6a-e3fa-4549-8183-819379712606.png)

## Proposal
Basically, KubeSphere DevOps take advantage of [Jenkins Pipeline](https://www.jenkins.io/doc/book/pipeline/).

ping @kubesphere/sig-devops 

/priority high
/area devops
